src/internal/components/pluggableVisualizations/pivotTable/PluggablePivotTable.tsx

```diff
@@ -599,6 +437,36 @@ export class PluggablePivotTable extends AbstractPluggableVisualization {
         return generateDimensions(mdObject, VisualizationTypes.TABLE);
     }
 
+    private adaptPropertiesToMdObject(
+        visualizationProperties: IVisualizationPropertiesWrapper,
+        mdObject: VisualizationObject.IVisualizationObjectContent,
+    ) {
+        // This is sanitization of properties from KD vs current mdObject from AD
+        const columnWidths = getColumnWidthsFromProperties(visualizationProperties);
+        if (columnWidths) {
+            this.sanitizeColumnWidths(columnWidths, mdObject);
+        }
+    }
+
+    private sanitizeColumnWidths(
+        columnWidths: ColumnWidthItem[],
+        mdObject: VisualizationObject.IVisualizationObjectContent,
+    ) {
+        const adaptedColumnWidths = adaptMdObjectWidthItemsToPivotTable(columnWidths, mdObject.buckets);
+        if (!isEqual(columnWidths, adaptedColumnWidths)) {
+            this.visualizationProperties.properties.controls.columnWidths = adaptedColumnWidths;
+            const { pushData } = this.callbacks;
+            pushData({
+                properties: {
+                    controls: {
+                        columnWidths: adaptedColumnWidths,
+                    },
+                },
+                sanitizedProperties: true,
```

Comment:
@ivanmjartan can you remember what this is needed for? I cannot see any usage in AD/KD, so not sure this is needed here...

Reply:
that could be removed

Reply:
Done